pages/coinbase/signin.py

```diff
@@ -63,12 +63,12 @@ def build_elements(self):
 
         self.email_field =          Field(self.driver, self.email_field_loc)
         self.pass_field =           Field(self.driver, self.pass_field_loc)
-        self.stay_signed_in_checkbox = ''
+        self.stay_signed_in_checkbox = CheckBox(self.driver, self.stay_signed_in_checkbox_loc)
```

Comment:
Don't make new templates. put tickets in for them.

Since we have no functionality for checkbox clicking, just leave it empty and make the ticket.